.changelog/104.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:breaking-change
+With the release of Go 1.17, Go 1.17 is now the lowest supported version of Go to use with terraform-plugin-framework.
```

Comment:
I'd recommend bumping the `go.mod` go directive via `go mod tidy -go=1.17`, since there are potentially some [module handling changes](https://golang.org/doc/go1.17#go-command). 👍